notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<p>same as before, <em>filtering/searching for one_hot features</em> instead of <em>update indirect missing data</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>